systemvm/debian/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -282,7 +281,10 @@ def set_backup(self):
 
         self._remove_ipv6_guest_gateway()
 
-        CsHelper.service("conntrackd", "restart")
+        if restart_conntrackd:
+            CsHelper.service("conntrackd", "restart")
+        else:
+            CsHelper.service("conntrackd", "stop")
```

Comment:
isn´t ignoring the contracd service here enough?

Reply:
@DaanHoogland 
it is better to stop it

Reply:
ok